### PR TITLE
Dev: some smaller margins for collapsed arrays on mobile.

### PR DESCRIPTION
**Pull request by tammoterhark**

### CODE REVIEW

**Comment by tammoterhark** [comment]
@shnoulle: why is the difference between: 

@media only screen and (max-device-width: 760px){ 

and

@media only screen and (max-width: 760px){ 

I prefer the last one: easier for testing on local, but I do not see the reason to add the "device". Please tell me.

File: templates/default/css/template.css [hunk 1]
```diff
@@ -654,15 +655,24 @@ div.yes-no .btn-group label {
   .table-bordered{
     border:none;
   }
-
-  .ls-answers tr:last-child{
+  
+  .ls-answers tbody:last-child tr:last-child{
```
Comment (Shnoulle) [inline templates/default/css/template.css]: Nee last child ? Why not all ?

**Comment by Shnoulle** [comment]
max-device-width : only for 'REAL screen can not be up to that'
max-width : real screen can be , but window of brownser is small

You'r right : we need max-width even after

File: templates/default/css/template.css (hunk 1 above)
Comment (tammoterhark) [inline templates/default/css/template.css]: There are multiple tbody statements in the the question (due to repeated headers?)

Comment (Shnoulle) [inline templates/default/css/template.css]: Yes: repeat header. But all need margin-bottom:0; no ? (and the last one can be great with  border-radius ;) )

**Comment by tpartner** [comment]
Also note that in some iPhones, max-device-width gives a wrong result when in landscape mode - it actually gives the portrait width. Best to stick to device-width.

**Comment by tpartner** [comment]
Oops - best to stick to max-width.

**Comment by Shnoulle** [comment]
Thanks @tpartner :) . 

I think it's better to use padding (like boostrap seems to do with btn) than min-height: what is your opinion here ?

**Comment by tpartner** [comment]
For responsive height? Yes, I use padding-top because you can use a percentage (of the parent) value:
height: 0;
min-height: 0;
padding-top: x%;

**Comment by Shnoulle** [comment]
? height: 0; no, i just mean : for a minimal height for 'pseudo-button' : something where user must put his finger to select :).

Missunderstanding ;)

**Comment by tpartner** [comment]
Oh, yeah, there padding is probably better than min-height.

**Pull request merged**